versioned_docs/version-22.10/api/clapi.md

```diff
@@ -2407,6 +2407,8 @@ Parameters that you can change are the following:
 | autologin\_key        | Used for auto login                                                                  |
 | template              | Name of the template to apply to the contact                                         |
 | timezone              | Timezone                                                                             |
+| reach_api             | Reach API Configuration                                                              |
+| reach_api_rt          | Reach API Realtime                                                                   |
```

Comment:
```suggestion
| reach_api_rt          | **1** if the user has access to the API realtime, **0** otherwise                                                                   |
```